### ISSUE TITLE
is it necessary to perform GCTA-COJO before using PolarMorphism software？

### ISSUE THREAD

**Issue by Moxxaaaa** [UMCUGenetics/PolarMorphism#9]
Hello Joanna,

Thanks for your software and article.

My team encountered some problems reproducing the results in your article (PMID: 35758773).

In https://github.com/UMCUGenetics/PolarMorphism/blob/master/template_for_github.Rmd, (line 28: print(paste0(wdir, trait, ".cojo.gz"))), I noticed that the input file The suffix is ".cojo.gz".

Does the example in the article pre-process the GWAS summary statistics raw data with GCTA-COJO?

If so, is it necessary to perform GCTA-COJO before using PolarMorphism software?

Thank you

Mo

**Comment by Joannavonberg** [comment]
Hi Mo,

Sorry for getting back to you so late! PolarMorphism was one of my PhD projects, and I have since started a postdoc in a different field so I can't spend a lot of time on it anymore.

To answer your question: no, that is not necessary! The reason the suffix of the input data I used is ".cojo.gz", is because these files were preprocessed to be run for conditional analysis and it was very convenient for me to use the same preprocessed files. They have been preprocessed as described in the paper, and COJO had not been run yet.
